File: gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/response/type/TenantKeyOne.kt
package gaia.sdk.response.type

import gaia.sdk.Uuid
import gaia.sdk.ISO8601
import gaia.sdk.Struct
import gaia.sdk.response.intf.*
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonCreator
import gaia.sdk.request.enumeration.*

/**
* This entity represents the output of a delete tenant impulse
*/
data class TenantKeyOne @JsonCreator constructor(
    @JsonProperty("tenantId") val tenantId:Uuid?, 
    @JsonProperty("reference") val reference:Uuid?
)

File: gaia-sdk-javascript/src/graphql/response/type/TenantKeyOne.ts


import {Uuid, ISO8601, Struct} from "../../GaiaClient";
import {Order} from "../../request/enumeration/Order";
import {OrderByField} from "../../request/enumeration/OrderByField";
import {EdgeOrderByField} from "../../request/enumeration/EdgeOrderByField";

/**
* This entity represents the output of a delete tenant impulse
*/
export interface TenantKeyOne {
    tenantId?:Uuid, 
    reference?:Uuid
}